ddl/ddl_worker.go

```diff
@@ -285,6 +286,10 @@ func (d *ddl) addBatchDDLJobs(tasks []*limitJobTask) {
 		if err != nil {
 			return errors.Trace(err)
 		}
+
+		// Drop table need to be supported when tikv is disk full, which depends on ddl job queue.
+		// So should let ddl job queue operations set.
+		txn.SetDiskFullOpt(kvrpcpb.DiskFullOpt_AllowedOnAlmostFull)
```

Comment:
If `DiskFullOpt_AllowedOnAlmostFull` is the default, why set it before executing?

Reply:
Default is DiskFullOpt_NotAllowedOnFull.
And operations with DiskFullOpt_AllowedOnAlmostFull will be dealt with when TiKV is disk full.

Reply:
Done, have cleared all the redundency set.